src/map/Pins.js

```diff
@@ -13,8 +13,28 @@
     Pins.prototype.constructor = Pins;
     Pins.prototype._class += " map_Pins";
 
+    
     Pins.prototype.publish("opacity", 1.0, "number", "Opacity", null, { tags: ["Advanced"] });
-    Pins.prototype.publish("fillColor", "#006ccc", "html-color", "Pin Color", null, { optional: true });
+    Pins.prototype.publish("fillColor", "#00FFDD", "html-color", "Pin Color", null, { optional: true });
+    
+    Pins.prototype.publish("strokeWidth", 1.5, "number", "Pin Border Thickness (pixels)", null, { tags: ["Basic"] });
+    Pins.prototype.publish("strokeColor", "#000000", "html-color", "Pin Border Color", null, { optional: true });
+    
+    Pins.prototype.publish("fontSize", 18, "number", "Font Size",null,{tags:["Basic","Shared"]});
+    Pins.prototype.publish("fontFamily", "FontAwesome", "string", "Font Name",null,{tags:["Basic","Shared","Shared"]});
+    Pins.prototype.publish("fontColor", "#000000", "html-color", "Font Color",null,{tags:["Basic","Shared"]});
+    
+    Pins.prototype.publish("pinType", "pin", "set", "Pin Type", ["pin","circle","rectangle","rectangle-pin"], { tags: ["Basic"] });
+    Pins.prototype.publish("arrowWidth", 18, "number", "Pin arrow width (pixels)", null, { tags: ["Basic"] });
+    Pins.prototype.publish("arrowHeight", 22, "number", "Pin arrow height (pixels)", null, { tags: ["Basic"] });
+    
+    Pins.prototype.publish("rectanglePinWidth", 50, "number", "Width of pin (pixels)", null, { tags: ["Basic"] });
+    Pins.prototype.publish("rectanglePinHeight", 50, "number", "Height of pin (pixels) (not including arrow)", null, { tags: ["Basic"] });
+    Pins.prototype.publish("rectangleCornerRadius", 10, "number", "Radius of rectangular pin corners (pixels)", null, { tags: ["Basic"] });
```

Comment:
Fantastic! I've been wishing for such a feature (assuming it hides disabled parameters in the property editor).